src/azul/service/elasticsearch_service.py

```diff
@@ -65,6 +65,9 @@ def __init__(self, missing_index: str):
         super().__init__(f'Index `{missing_index}` was not found')
 
 
+SourceFilters = List[str]
```

Comment:
Corresponding commit title could be improved. You're not just moving the alias. And it's *alias*, not *type definition*. https://docs.python.org/3/library/typing.html#type-aliases

Reply:
I renamed that commit, but the rest of the changes are fixups.